packages/mdl-button/index.scss

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
```

Comment:
We can't rely on ripple (JS) though, so the active state modifier is still extremely valuable.

I was planning on handling the transition when doing the shadows properly after elevation is added. Since I also need to handle it for desktop not having elevation by default with that.

Focus state, once again anything handled by JS should be additive to this. For a base implementation that doesn't rely on JS this is a good first-pass. We can add a button component later to enhance when that is available for users.

Disabled raised button backgrounds do change. Button Elevation section, underneath the image:
> Disabled text: 30% #FFFFFF
> Disabled button: 12% #FFFFFF

I'm not seeing any active state things affecting my disabled buttons here. I don't mind making sure it doesn't, however I don't see any actual problem currently. Could you point out why please?

Ah, yea. Need to test cross-browser. I did toggle the `-webkit-appearance` in dev, but it had no visual affect. Anything in particular setting that to `none` causes?